package.json

```diff
@@ -13,5 +13,8 @@
         "1.1.0": "provideLinter"
       }
     }
+  },
+  "dependencies": {
+    "atom-linter": "^3.2.0"
```

Comment:
Nope, nada. I have several offenses on the file (run from the command line), but this does not appear to run. Even if I remove rubocop path I get no error, so maybe it's not hooked up to linter or something